resources/views/auth/passwords/email.html.twig

```diff
@@ -10,127 +10,76 @@
 {% block title %}{{ forgot_password.title }}{% endblock %}
 
 {% block body %}
-
-<section
-    class="auth"
-    style="background-image: url('/images/bg_crowd.jpg');">
-
-    <div
-        class="auth-email">
-
-        <a
-            class="auth-return-link"
-            href="/"
-            title="Return Home">
-            <i class="fas fa-chevron-left"></i>{{ forgot_password.return }}
-        </a>
-
-        <div
-            class="auth-heading">
-
-            <h1>{{ forgot_password.header }}</h1>
-
-        </div>
-
-        <form
-            action="{{ routes.password.email }}"
-            class="form__wrapper"
-            method="POST">
-
-            {{ csrf_field() }}
-
-            <div
-                class="auth-content">
-
-                <div
-                    class="auth-scroll-wrapper">
-
-                    <div
-                        class="auth-copy">
-
-                        <div
-                            class="flex-grid">
-
-                            {% if session('status') is not empty %}
-                                <div class="box full">
-                                    <span class="auth-success">
-                                        {{ session('status') }}
-                                    </span>
-                                </div>
-                            {% endif %}
-
-                            {% if errors.has('email') %}
-                                <div class="box full">
-                                    <span class="auth-alert">
-                                        <strong>{{ errors.first('email') }}</strong>
-                                    </span>
+    <div data-clone>
+        <section class="tc-auth" data-c-padding="tb(triple)" data-c-overlay="black(80)" style="background-image: url('/images/bg_crowd.jpg');">
+            <div class="tc-auth-wrapper" data-c-padding="tb(triple)" data-c-container="small">
+                {# Return to the homepage link. #}
+                <a data-c-color="white" href="/" title="Return Home">
```

Comment:
"Return Home" link should go to the Manager Home page.

Reply:
Same with `login.html.twig`, `register.html.twig`

Reply:
👍